runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
Yeah I saw that PR after it was pointed out in Gitter.  I agree with your sentiment in hindsight.  But, before `snapshot_util()` was pointed out to me, screen testing was a little bit difficult to comprehend. The output for highlights that aren't configured made it even more confusing.

At the time of writing, my view was that if I need to test a particular highlight, I only want to focus on the appearance of that highlight and more broad tests for highlighting would cover the ones I wasn't testing.  For this PR in particular, I didn't want to care about the `qf` syntax highlighting for line numbers and filenames.

I actually think that the output of `snapshot_util()` should be what's displayed for failed test, but followed with a list of the highlights instead of them being inline with the screen text.